services/frontend-service/Earthfile

```diff
@@ -116,4 +116,4 @@ BUILD_ALL:
     BUILD +release --registry=$registry --tag=$tag
```

Comment:
you need to add `fix:` as prefix of the commit message to make the linter happy  and there's a failing test. Apart from this, looks good to me.